### ISSUE TITLE
Test "not find source-analysis <whitespace-at-eol>" fail (on Debian)

### ISSUE THREAD

**Issue by lkppo** [metaeducation/ren-c#961]
(not find source-analysis <whitespace-at-eol>) "failed, test returned #[false]"

**Comment by hostilefork** [comment]
Impressed that you're running the tests!  Great!  Since you are running them you presumably found the README.md, which is probably out of date:

https://github.com/metaeducation/ren-c/blob/master/tests/README.md

Any improvements you can make are welcome.

You've discovered here that I don't *always* wait for the source analysis when committing.  In particular, if I'm merging things.  The odd whitespace error does creep in as a result, and gets patched up when it does.  They are not considered super high priority errors, just don't want them to stick around.

I'd like to set things up so that the tests are run on some CI.  Shixin put things on AppVeyor and had tests for a bit, but that atrophied at some point and he hasn't been around.  There's been some talk of him coming back, though...he may be at the Philadelphia conference in July.  *(Probably would be a bit of a trip for you from France, but if you are looking for a reason to take a vacation, 4th of July weekend!)*

I'm not really ready to treat whitespace-at-end-of-line as a GitHub-issues-level-error yet.  So maybe the thing to do is to move these to some kind of warning category, so people don't report it.

**Comment by lkppo** [comment]
Yes I read it, and it is not up to date but I didn't see it since I just wanted to launch tests after I read your slides _Ren-C 2016 / Atronix_. The trip is a bit expensive. I would need more than one talk to get this far. 

I rented a cheap server under OpenBSD to host me. I had the idea to set up an automatic build and daily tests since it is not possible otherwise. We could do the same under Debian.
